intermediate_source/inductor_debug_cpu.py

```diff
@@ -0,0 +1,617 @@
+
+for _ in range(10):
+    model(**input_dict)  # compiled_model(**input_dict) to get inductor model profiling
+
+total = 0
+with profile(
+    activities=[ProfilerActivity.CPU],
+    schedule=my_schedule,
+    on_trace_ready=trace_handler
+) as p:
+    for _ in range(100):
+        model(**input_dict)  # compiled_model(**input_dict) to get inductor model profiling
+        p.step()
+
+######################################################################
+# We get the following performance profiling table for the eager-mode model (omitting some columns):
+#
+# .. code-block:: shell
+#
+#     -------------------------  ------------  ------------  ------------  ------------  
+#                          Name   CPU total %     CPU total  CPU time avg    # of Calls  
+#     -------------------------  ------------  ------------  ------------  ------------  
+#                   aten::addmm        45.73%     370.814ms       1.024ms           362  
+#                     aten::add        19.89%     161.276ms     444.287us           363  
+#                   aten::copy_        14.97%     121.416ms     248.803us           488  
+#                     aten::mul         9.02%      73.154ms     377.082us           194  
+#               aten::clamp_min         8.81%      71.444ms     744.208us            96  
+#                     aten::bmm         5.46%      44.258ms     922.042us            48  
+#                 ProfilerStep*       100.00%     810.920ms     810.920ms             1  
+#                     aten::div         2.89%      23.447ms     976.958us            24  
+#                aten::_softmax         1.00%       8.087ms     336.958us            24  
+#                  aten::linear        46.48%     376.888ms       1.041ms           362  
+#                   aten::clone         2.77%      22.430ms     228.878us            98  
+#                       aten::t         0.31%       2.502ms       6.912us           362  
+#                    aten::view         0.14%       1.161ms       1.366us           850  
+#               aten::transpose         0.17%       1.377ms       3.567us           386  
+#            aten::index_select         0.12%     952.000us     317.333us             3  
+#                  aten::expand         0.12%     986.000us       2.153us           458  
+#                  aten::matmul         8.31%      67.420ms       1.405ms            48  
+#                     aten::cat         0.09%     703.000us     703.000us             1  
+#              aten::as_strided         0.08%     656.000us       0.681us           963  
+#                    aten::relu         8.86%      71.864ms     748.583us            96  
+#     -------------------------  ------------  ------------  ------------  ------------  
+#     Self CPU time total: 810.920ms
+#
+
+######################################################################
+#
+# Similarly, we also get the table for the compiled model with Inductor (omitting some columns):
+#
```

Comment:
This will be generated in the final [HTML]( I see duplicate outputs in the resulting HTML: https://docs-preview.pytorch.org/pytorch/tutorials/2430/intermediate/inductor_debug_cpu.html#performance-profiling ). Can we remove this block or state in which env you get this numbers if it's important to show specifically these results?

Reply:
Hi, @svekars. The generated table is not in good format ( It's too long for 1 line).
For all `output` related blocks, can we choose the comment out `print` and use our results ( with env setting and system types)?

Reply:
yes, I think it is fine. Or we can keep both and say something like this before the not generated output: The following output was generated on .... hardware.